### PR TITLE
init: add the Eclipse OSD implementation of the Open System Descripti…

### PR DESCRIPTION
**Pull request by klopes25**
Eclipse OSD is the implementation of the Open System Description format with the Eclipse Modeling Framework.

### CODE REVIEW

**Review by fthomasfr — APPROVED** [review]

File: .mvn/extensions.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<extensions>
+  <extension>
+    <groupId>org.eclipse.tycho.extras</groupId>
+    <artifactId>tycho-pomless</artifactId>
+    <version>1.2.0</version>
+  </extension>
+</extensions>
```
Comment (jeanguyomarch) [inline .mvn/extensions.xml]: Is it normal not to have a newline at the end of the file?

File: documentation/SampleProject/model/sampleProject.ecore [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jeanguyomarch) [inline documentation/SampleProject/model/sampleProject.ecore]: Isn't it a generated file? Generated files should not be committed to the repository.

**Review by jeanguyomarch — CHANGES_REQUESTED** [review]
To sum-up the key points of our discussion:
- the documentation generated from the javadoc should not be committed in this repository. Once travis is setup, it could automatically build the doc for us.
- We should make sure that files end with a newline (otherwise, git usually complains about these files).
- "Huge" binary files (such as videos or large images) should be stored using [Git LFS](https://git-lfs.github.com/). [This tutorial](https://help.github.com/en/articles/configuring-git-large-file-storage) should provide hints to do so.
- A [code of conduct](https://github.com/krono-safe/vim-asterios/blob/master/CODE_OF_CONDUCT.md) would be nice to add.
- Jenkinsfiles should not be committed here, as they are inherent to an internal Jenkins system. Ultimately, they will be replaced with something like `.travis.yml`.

**Comment by jeanguyomarch** [comment]
Also, the commit message is not properly written. This causes github to truncate the message, which is quite disgracious. You may want to follow [this article](https://thoughtbot.com/blog/5-useful-tips-for-a-better-commit-message).

File: documentation/SampleProject/model/sampleProject.ecore (hunk 2 above)
Comment (klopes25) [reply documentation/SampleProject/model/sampleProject.ecore]: The generated files are no longer in the repository.

File: .mvn/extensions.xml (hunk 1 above)
Comment (klopes25) [reply .mvn/extensions.xml]: No, it's changed.

**Comment by jeanguyomarch** [comment]
I am not competent to judge the Java part. The rest looks good to me. Any additional thoughs @superwhoopy?

**Review by jeanguyomarch — APPROVED** [review]

**Review by superwhoopy — APPROVED** [review]
Champagne ! 🍾 
:-)

**Pull request merged**